packages/synapse-react-client/src/utils/SynapseConstants.ts

```diff
@@ -134,15 +134,30 @@ export const URL_TERMS_CONDITIONS_AGREEMENT =
 /** Class names */
 export const LOGIN_BACK_BUTTON_CLASS_NAME = 'login-flow-back-button'
 
+/* Login methods */
+export const LOGIN_USERNAME_PASSWORD = 'EMAIL'
+export const LOGIN_OAUTH2_GOOGLE = 'GOOGLE'
+export const LOGIN_OAUTH2_ORCID = 'ORCID'
+
 export const OAUTH2_PROVIDERS = {
-  GOOGLE: 'GOOGLE_OAUTH_2_0',
-  ORCID: 'ORCID',
+  [LOGIN_OAUTH2_GOOGLE]: 'GOOGLE_OAUTH_2_0',
+  [LOGIN_OAUTH2_ORCID]: 'ORCID',
```

Comment:
OAUTH2_PROVIDERS keys are used in SageAccountWeb when mapping provider query params to login method -- use the constants so that the keys are consistent here and in other components (LoginMethodButton, LastLoginInfo)